### DISCUSSION TITLE
use custom variable

### DISCUSSION THREAD

**Discussion by og900aero** [JanDeDobbeleer/oh-my-posh#5875] [Troubleshoot] [answered]
Hello,

For file manager I use lf file manager. It has a $LF_LEVEL variable, which I used in .bashrc:
[ -n "$LF_LEVEL" ] && PS1="(lf) $PS1"

Now that I am using oh my posh, how can I set this? I tried the below but I get an error message:
```
"segments": [
{
"type": "text",
"style": "plain",
"foreground": "#FFFFFFFF",
"template":"{{ if eq .Env.LF_LEVEL 1 }}╭─(lf)─{{ else }}╭─{ end }}"
},
```

```
: error calling eq: incompatible types for comparison   shyciii@debian  /home/Data  took 0ms 
╰─$ 
```

**Comment by JanDeDobbeleer** [comment, marked as answer]
@og900aero environment variables are always strings (also in the shell), so you need to compare against a string:

```json
"template":"{{ if eq .Env.LF_LEVEL \"1\" }}╭─(lf)─{{ else }}╭─{ end }}"
```

**Reply by og900aero** [reply]
Thx, working great!
